### ISSUE TITLE
Tooltips Dark & Light - Accessibility update

### ISSUE THREAD

**Issue by cathystoli** [chanzuckerberg/sci-components#704]
**Issue:** 1. Focus indicator for the icon button with a tooltip is a wide oval shape, which is the wrong shape.

**Solution:** 1. Focus indicator should be a circle around the icon button.
